packages/dynamic_layouts/.gitignore

```diff
@@ -9,6 +9,7 @@
 .history
 .svn/
 migrate_working_dir/
+example/
```

Comment:
Why? We probably want to check the example in.

Reply:
It's not finished yet, and I wasn't sure if it should be included in this PR or in another one

Reply:
A follow up PR is probably best. 👍

Reply:
Yeah, it should be a separate PR, but we also shouldn't check in a gitignore that ignores changes to examples.